app/models/loader/orchestrate.rb

```diff
@@ -306,13 +292,23 @@ def insert_policy_log_records(table)
             kind, 
             subject)
           SELECT
-          #{db.literal(policy_id)},
-          #{db.literal(policy_version.version)},
-          'INSERT',
-          '#{table}'::policy_log_kind,
-          slice(hstore(#{table}), #{db.literal(primary_key_columns)})
+            r.policy_id,
+            r.version,
+            r.operation,
+            r.kind,
+            r.subject
           FROM
-          #{schema_name}.#{table}
+            #{schema_name}.#{table},
```

Comment:
This doesn't look right. Why are we joining the table with the rows returned from the function?